packages/odyssey-storybook/.storybook/manager-head.html

```diff
@@ -1,5 +1,6 @@
 <style>
-  .sidebar-item[id$="-dont"] {
+  .sidebar-item[id$="-dont"],
```

Comment:
As noted in the previous PR, this bit of ugliness is required until SB allows decorators (or a `hidden` prop) for Sidebar items.